packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/configuration/RequestLoggingResponseFilter.java

```diff
@@ -16,6 +18,12 @@ public class RequestLoggingResponseFilter implements ContainerResponseFilter {
 	public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext) throws IOException {
 		final String user = requestContext.getSecurityContext().getUserPrincipal() != null ? requestContext.getSecurityContext().getUserPrincipal().getName() : "Anonymous";
 		final long durationMs = Instant.now().toEpochMilli() - (long)requestContext.getProperty(RequestLoggingRequestFilter.REQUEST_START_TIMESTAMP_MS);
-		log.info("REQUEST COMPLETED | " + requestContext.getRequest().getMethod() + " | " + requestContext.getUriInfo().getRequestUri() + " | " + user + " | " + durationMs + "ms");
+		log.info(JsonString.write(Map.of(
+			"log_message", "REQUEST COMPLETED",
```

Comment:
These same fields show up in a few places.  To avoid mistakes that could affect detecting these logs (`log_message` vs `logMessage`) should we through these into an enum or just static Strings somewhere to access?

Reply:
I moved this into a `StructuredLog` application scoped service. This will cover most cases but there may be instances where we need structured logs and this is not `@Inject`able in which case we'll have to rely on convention and uses the bare `log.info` function.